CMakePresets.json

```diff
@@ -10,14 +10,6 @@
         "CMAKE_CXX_STANDARD": "17"
       }
     },
-    {
-      "name": "_debug-base",
-      "hidden": true,
-      "cacheVariables": {
-        "CMAKE_BUILD_TYPE": "Debug",
-        "CMAKE_CXX_FLAGS": "-fsanitize=address -fsanitize=pointer-compare -fsanitize=pointer-subtract -fsanitize=leak -fsanitize=undefined"
-      }
-    },
     {
```

Comment:
if we still adopt #76, `_debug-base` could share `BEMAN_BUILDSYS_SANITIZER=ASan`. 

Just want to slightly nudge this a little bit.

Otherwise okay I can bring it back. I don't really see the value of of `_debug-base`/ `_release-base` if we only have `CMAKE_BUILD_TYPE` in it but I do see the argument in extensibility.

Reply:
`_debug-base` is back.